README.md

```diff
@@ -150,6 +150,56 @@ co(function * () {
 
 ```
 
+### Shared Secret Example
```

Comment:
We should also update the description of this module at the top of the README to talk about PSK, how to use that, and how it's different from IPR